[2025-07-09T07:58:23.779+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-09T07:58:23.811+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_airflare_data.extract manual__2025-07-09T07:58:17.397989+00:00 map_index=20 [queued]>
[2025-07-09T07:58:23.820+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_airflare_data.extract manual__2025-07-09T07:58:17.397989+00:00 map_index=20 [queued]>
[2025-07-09T07:58:23.821+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-07-09T07:58:23.836+0000] {taskinstance.py:2330} INFO - Executing <Mapped(_PythonDecoratedOperator): extract> on 2025-07-09 07:58:17.397989+00:00
[2025-07-09T07:58:23.849+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=7214) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-07-09T07:58:23.851+0000] {standard_task_runner.py:63} INFO - Started process 7236 to run task
[2025-07-09T07:58:23.851+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'ETL_airflare_data', 'extract', 'manual__2025-07-09T07:58:17.397989+00:00', '--job-id', '620', '--raw', '--subdir', 'DAGS_FOLDER/Untitled-1.py', '--cfg-path', '/tmp/tmpxyi8ops4', '--map-index', '20']
[2025-07-09T07:58:23.852+0000] {standard_task_runner.py:91} INFO - Job 620: Subtask extract
[2025-07-09T07:58:23.906+0000] {task_command.py:426} INFO - Running <TaskInstance: ETL_airflare_data.extract manual__2025-07-09T07:58:17.397989+00:00 map_index=20 [running]> on host e6a32b72ede0
[2025-07-09T07:58:24.003+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ETL_airflare_data' AIRFLOW_CTX_TASK_ID='extract' AIRFLOW_CTX_EXECUTION_DATE='2025-07-09T07:58:17.397989+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-09T07:58:17.397989+00:00'
[2025-07-09T07:58:24.005+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-09T07:58:24.006+0000] {Untitled-1.py:134} INFO - Начало извлечения данных для направления IKT-SGN
[2025-07-09T07:58:24.778+0000] {Untitled-1.py:59} INFO - Создание подключения к S3
[2025-07-09T07:58:24.780+0000] {Untitled-1.py:62} INFO - Подключение установлено
[2025-07-09T07:58:24.792+0000] {base.py:84} INFO - Using connection ID 'minio' for task execution.
[2025-07-09T07:58:24.793+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='minio', conn_type='aws') credentials retrieved from login and password.
[2025-07-09T07:58:25.131+0000] {Untitled-1.py:68} INFO - Загрузка данных в S3
[2025-07-09T07:58:25.210+0000] {Untitled-1.py:79} INFO - Загрузка завершена успешно
[2025-07-09T07:58:25.211+0000] {Untitled-1.py:147} INFO - Данные успешно сохранены во временное хранилище
[2025-07-09T07:58:25.212+0000] {Untitled-1.py:148} INFO - Данные успешно извлечены и переданы в Xcom
[2025-07-09T07:58:25.213+0000] {python.py:237} INFO - Done. Returned value was: {'path': {'bucket': 'airflare', 'key': 'temp/response_IKT_SGN.json'}, 'rout': {'origin': 'IKT', 'destination': 'SGN', 'bd': 'Иркутск_Хошимин_outbound'}}
[2025-07-09T07:58:25.214+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-09T07:58:25.234+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ETL_airflare_data, task_id=extract, run_id=manual__2025-07-09T07:58:17.397989+00:00, map_index=20, execution_date=20250709T075817, start_date=20250709T075823, end_date=20250709T075825
[2025-07-09T07:58:25.268+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-09T07:58:25.299+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-09T07:58:25.301+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
